client.js

```diff
@@ -217,20 +206,49 @@ async function selectCountriesAndCities(hzClient) {
     console.log("--------------------");
 }
 
+async function nonStopMapExample(map) {
```

Comment:
Please add the comment in Java sample here and this function should accept a client instance instead of the map instance.